bindings/wasm/src/account/update/create_service.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use js_sys::Promise;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen_futures::future_to_promise;
+
+use identity::account::Update;
+use identity::account::UpdateError::MissingRequiredField;
+use identity::core::Url;
+use identity::did::ServiceEndpoint;
+
+use crate::account::wasm_account::WasmAccount;
+use crate::error::wasm_error;
+use crate::error::Result;
+use crate::error::WasmResult;
+
+#[wasm_bindgen(js_class = Account)]
+impl WasmAccount {
+  /// Adds a new Service to the DID Document.
+  #[wasm_bindgen(js_name = createService)]
+  pub fn create_service(&mut self, options: &CreateServiceOptions) -> Result<Promise> {
+    let account = self.0.clone();
+
+    let service_type: String = match options.serviceType() {
+      Some(value) => value,
+      None => return Err(wasm_error(MissingRequiredField("serviceType"))),
+    };
+
+    let fragment = match options.fragment() {
+      Some(value) => value,
+      None => return Err(wasm_error(MissingRequiredField("fragment"))),
+    };
+
+    let endpoint = match options.endpoint() {
+      Some(v) => v,
+      None => return Err(wasm_error(MissingRequiredField("endpoint"))),
+    };
+
+    let endpoint = Url::parse(endpoint.as_str()).wasm_result()?;
+
+    let update = Update::CreateService {
+      fragment,
+      type_: service_type,
+      endpoint: ServiceEndpoint::from(endpoint),
+      properties: None, //ToDo
+    };
```

Comment:
TODO properties. What should we actually pass in for `properties`, just `JsValue`/`any`? That's basically the equivalent...